public/scripts/admin/subcomponents/OrgForm.jsx

```diff
@@ -42,7 +54,24 @@ var OrgForm = React.createClass({
       }.bind(this)
     });
   },
+  onSelect: function(option) {
+    this.setState({selected: option});
+  },
   render: function() {
+    if (!this.props.org.name) {
```

Comment:
Will this work for the Add Organization part of this form? this.props.org.name will be undefined.